doc: Document batch insertion error

Ben-PH opened the pull request:
<!-- Please refer to our CONTRIBUTING documentation for any questions on submitting a pull request. -->
<!-- Provide a general summary of your changes in the Title above. -->

## Description

<!-- Describe your changes in detail. -->
<!-- You may want to answer some of the following questions: -->
<!-- What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...) -->
<!-- What is the current behavior?** (You can also link to an open issue here) -->
<!-- What is the new behavior (if this is a feature change)? -->
<!-- Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?) -->

## Related Issue(s)

<!-- This project accepts pull requests related to open issues. -->
<!-- If suggesting a new feature or change, please discuss it in an issue first. -->
<!-- If fixing a bug, there should be an issue describing it with steps to reproduce. -->
<!-- Please link to the issue(s) here -->

<!-- Closes # -->
<!-- Fixes # -->

## Other information

<!-- Any other information that is important to this PR such as screenshots of how the component looks before and after the change. -->
<!-- Feel free to remove this section if you will not use it. -->

## Checklist

<!-- Please check if the PR fulfills these requirements. -->

- [ ] I have read and understand the [contributor guidelines](https://github.com/privacy-scaling-explorations/zk-kit.rust/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/privacy-scaling-explorations/zk-kit.rust/blob/main/CODE_OF_CONDUCT.md).
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have run `yarn style` without getting any errors
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

> [!IMPORTANT]
> We do not accept minor grammatical fixes (e.g., correcting typos, rewording sentences) unless they significantly improve clarity in technical documentation. These contributions, while appreciated, are not a priority for merging. If there is a grammatical mistake, please feel free to message the team.

Ben-PH commented:
Just a minor documentation addition that shows an empty batch insert will error.

vplasencia reviewed (approved):
Thank you very much for this PR! 🚀

vplasencia reviewed (changes requested):
Hey @Ben-PH! Could you run `make fmt` to fix the formatting before we merge?

Ben-PH commented:
I added cmake and got a dprint error. I added dprint then got this error:

```
[dprint-plugin-exec]
Could not start dynamically linked executable: /home/ben/.cache/dprint/cache/plugins/dprint-plugin-exec/0.4.4/x86_64/dprint-plugin-exec
NixOS cannot run dynamically linked executables intended for generic
linux environments out of the box. For more information, see:
https://nix.dev/permalink/stub-ld
[dprint-plugin-prettier]
Could not start dynamically linked executable: /home/ben/.cache/dprint/cache/plugins/dprint-plugin-prettier/0.40.0/x86_64/dprint-plugin-prettier
NixOS cannot run dynamically linked executables intended for generic
linux environments out of the box. For more information, see:
https://nix.dev/permalink/stub-ld
[dprint-plugin-exec]
Could not start dynamically linked executable: /home/ben/.cache/dprint/cache/plugins/dprint-plugin-exec/0.4.4/x86_64/dprint-plugin-exec
NixOS cannot run dynamically linked executables intended for generic
linux environments out of the box. For more information, see:
https://nix.dev/permalink/stub-ld
[dprint-plugin-prettier]
Could not start dynamically linked executable: /home/ben/.cache/dprint/cache/plugins/dprint-plugin-prettier/0.40.0/x86_64/dprint-plugin-prettier
NixOS cannot run dynamically linked executables intended for generic
linux environments out of the box. For more information, see:
https://nix.dev/permalink/stub-ld
Error resolving plugin https://plugins.dprint.dev/exec-0.4.4.json: Failed plugin schema verification. This may indicate you are using an old version of the dprint CLI or plugin and should upgrade: Could not read success response.: failed to fill whole buffer
make: *** [Makefile:38: fmt] Error 12
```

I'm happy for this issue to be closed without merging. It's a trivial thing on its own, and there's a lot of other warnings missing IIRC.
